app/services/movies_service.rb

```diff
@@ -11,15 +11,13 @@ def search(keyword)
 
   def top_rated
     response = connection.get("3/movie/top_rated")
-    JSON.parse(response.body)["results"]
+    JSON.parse(response.body, symbolize_names: true)["results"]
```

Comment:
Adding the symbolize_names here breaks my code that's now on main so I'll need to make changes to what I wrote to make this work if we decide to keep it.